pom.xml

```diff
@@ -193,7 +193,7 @@
         <!-- JSON Schema Validator -->
         <org.everit.json.schema.version>1.14.1</org.everit.json.schema.version><!-- TODO unification -->
         <jackson-datatype-json-org.version>2.13.4</jackson-datatype-json-org.version>
-        <jackson-dataformat-yaml.version>2.14.0</jackson-dataformat-yaml.version>
+        <jackson-dataformat-yaml.version>2.13.4</jackson-dataformat-yaml.version>
```

Comment:
The native image was failing to build because of this dependency, we should probably look closer at the issue.